api/app/signals/apps/api/views/signal_context.py

```diff
@@ -34,13 +38,26 @@ def get_queryset(self, *args, **kwargs):
         return Signal.objects.filter_for_user(user=self.request.user)
 
     def geography(self, request, pk=None):
+        signal = self.get_object()
+
+        signals_for_geography_qs = Signal.objects.annotate(
+            distance_from_point=Distance('location__geometrie', signal.location.geometrie),
+        ).filter(
+            (Q(parent__isnull=True) & Q(children__isnull=True)) | Q(parent__isnull=False),
+            distance_from_point__lte=app_settings.SIGNAL_CONTEXT_GEOGRAPHY_RADIUS,
+            category_assignment__category_id=signal.category_assignment.category.pk,
+            created_at__gte=(
+                timezone.now() - timezone.timedelta(weeks=app_settings.SIGNAL_CONTEXT_GEOGRAPHY_CREATED_DELTA_WEEKS)
+            ),
+        ).exclude(pk=signal.pk)
```

Comment:
We include the signal itself for the reporter context.

Reply:
Correct, however in the design it is stated that it must exclude the signal itself in this context. Therefore I excluded it, the other option is to include it and then the FE should not show it. @tcoenen what do you think that we should do? Lets have a small talk about this.

Reply:
we agreed to leave it --- consistent with UX design, not entirely consistent with the context reporter endpoint